benefits/enrollment/views.py

```diff
@@ -163,26 +163,43 @@ def success(request):
     """View handler for the final success page."""
     request.path = "/enrollment/success"
     session.update(request, origin=reverse(ROUTE_SUCCESS))
+
     verifier = session.verifier(request)
-    icon = viewmodels.Icon("bankcardcheck", pgettext("image alt text", "core.icons.bankcardcheck"))
-    page = viewmodels.Page(
-        classes="no-image-mobile",
-        title=_("enrollment.pages.success.title"),
-        headline=_("enrollment.pages.success.headline"),
+
+    success_item = viewmodels.MediaItem(
```

Comment:
I noticed this type of refactor wasn't applied to the dicts in [`eligibility` `start`](https://github.com/cal-itp/benefits/blob/dev/benefits/eligibility/views.py#L85-L104), but then I remembered about the bullet points. Just wanted to document/confirm that that's why we're leaving those alone for now.

Reply:
See #996